sklearn/model_selection/tests/test_search.py

```diff
@@ -642,41 +642,53 @@ def test_grid_search_results():
     params = [dict(kernel=['rbf', ], C=[1, 10], gamma=[0.1, 1]),
               dict(kernel=['poly', ], degree=[1, 2])]
     grid_search = GridSearchCV(SVC(), cv=n_splits, iid=False,
-                               param_grid=params)
+                               param_grid=params, return_train_score=True)
```

Comment:
Ah good catch. It should be false by default... Not many would want to see training score...